### PR TITLE
Allow changing legacy_origin

### PR DESCRIPTION
**Pull request by mhaskel**
Sometimes you don't want to use the default from params.pp

### CODE REVIEW

**Comment by mhaskel** [comment]
Fixes #423

File: README.md [hunk 1]
```diff
@@ -277,6 +278,7 @@ apt::sources:
 
 ####apt::unattended_upgrades
 
+* `legacy_origin`: If set to true, use the old `Unattended-Upgrade::Allowed-Origins` variable. If false, use `Unattended-Upgrade::Origins-Pattern`. OS-dependent defaults are defined in `apt::params`.
```
Comment (hunner) [inline README.md]: What to mention the default?

**Comment by cyberious** [comment]
Can we acceptance test this scenario?

**Comment by mhaskel** [comment]
@cyberious I don't see why this needs acceptance test, the only thing it's changing is a file, which is unit-tested

**Pull request merged**